sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/TestUtils.java

```diff
@@ -31,15 +32,24 @@
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 public class TestUtils {
+    private static final ClientLogger LOGGER = new ClientLogger(TestUtils.class);
+
     // System and application properties from the generated test message.
     static final Instant ENQUEUED_TIME = Instant.ofEpochSecond(1561344661);
     static final Long SEQUENCE_NUMBER = 1025L;
     static final String OTHER_SYSTEM_PROPERTY = "Some-other-system-property";
     static final Boolean OTHER_SYSTEM_PROPERTY_VALUE = Boolean.TRUE;
     static final Map<String, Object> APPLICATION_PROPERTIES = new HashMap<>();
+    static final int USE_CASE_DEFAULT = 0;
```

Comment:
Do these need to be variables here? Can they live in their respective test cases. Some of these are only used once.

Reply:
If these numbers lie in their respective test files then another test file can use same number again without knowing it.
Putting here, gives next developer visibility to ensure that same index number is not reused by mistake.